patternfly/patternfly-elements
# fix: pfe tabs padding update

## <Pull request description>

Updating padding for tabs.
- https://rally1.rallydev.com/#/270861059696d/teamboard?detail=%2Fuserstory%2F492652479940%2Ftasks&view=113b0e72-0781-41c7-b6d6-e161d3349c72
- https://xd.adobe.com/view/cc7627a7-ead9-4c7a-8664-ac526c680bd9-9925/specs/. 
- https://trello.com/c/2zvfhO7K/847-fix-pfe-tabs-padding
- Update variables to new standard with fallbacks in pfe-tabs as well as update some other code while I am in there.


### Preview


Link(s) to demo page(s) where this element can be viewed:
- https://deploy-preview-1457--happy-galileo-ea79c4.netlify.app/elements/pfe-tabs/demo/


### What has changed and why



- pfe-tabs.scss has been updated with new padding vars and values as well as some other updates. 


### Testing instructions



1. Open the tabs demo page and you should see the following changes in the XD file (purple spacer is 48px by 48px)
Demo page
- 

XD file:
- https://xd.adobe.com/view/cc7627a7-ead9-4c7a-8664-ac526c680bd9-9925/specs/

Test style overrides via the `inspector-stylesheet`
paste in

```
:root {
    --pfe-tabs__panel--PaddingTop: 100px;
    --pfe-tabs__panel--PaddingRight: 100px;
    --pfe-tabs__panel--PaddingBottom: 100px;
    --pfe-tabs__panel--PaddingLeft: 100px;

    --pfe-tabs__panel--BorderTop: 4px dashed purple;
    --pfe-tabs__panel--BorderRight: 4px dashed purple;
    --pfe-tabs__panel--BorderLeft: 4px dashed purple;
    --pfe-tabs__panel--BorderBottom: 4px dashed purple;

    --pfe-tabs__panel--BackgroundColor: pink;
}
```

Make sure overrides work and default styles are as expected from mockups. Also make sure there is not duplicate padding (panel and container)


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
